pytest/common_lib/shared/__init__.py

```diff
@@ -376,53 +415,68 @@ def start_cluster_with_mpc(
     (key, nonce) = cluster.contract_node.get_key_and_nonce()
     create_txs = []
     access_txs = []
-    mpc_nodes: List[MpcNode] = []
-    pytest_keys_per_node = []
-    secondary_near_account: Optional[NearAccount] = None
+    mpc_nodes: list[MpcNode] = []
+    pytest_keys_per_node = [[] for _ in range(len(configs))]
```

Comment:
Nit:
This feels more "pythonic"
```suggestion
    pytest_keys_per_node = [[] for _ in configs]
```